project/backend/src/routes/notes.js

```diff
@@ -0,0 +1,160 @@
+import express from 'express';
+import { PrismaClient } from '../generated/prisma/index.js';
+
+const router = express.Router();
+const prisma = new PrismaClient();
+
+// GET get all notes
+router.get("/", async (req, res) => {
+  try {
+    const notes = await prisma.note.findMany({
+      orderBy: {
+        createdAt: "desc",
+      },
+    });
+    res.json(notes);
+  } catch (error) {
+    console.error("Error fetching notes:", error);
+    res.status(500).json({ error: "Failed to fetch notes" });
+  }
+});
```

Comment:
@Alaames for the routes, it' always better to keep onnly the route definition and put the controllers in a separated folder like this :
```suggestion
router.get("/", getNotes);
```

and import `getNotes` directly from a separated controller folder